proto-build.js

```diff
@@ -0,0 +1,27 @@
+const Axios = require('axios');
+const shell = require('shelljs');
+const fs = require('fs');
+
+const githubBranch = process.env.GITHUB_BRANCH || 'master';
+
+/**
+ * this function is called before build and gets the proto file from the spike-service in github, and generate it to node and typescript.
+ * @param {*} fileName - The name of the proto file
+ * @param {*} protoPath - The path to the proto in github
+ */
+async function getSpikeProto(fileName, protoPath) {
+    try {
+        const proto = await Axios.get(`https://raw.githubusercontent.com/meateam/spike-service/${githubBranch}/proto/spike-service/${fileName}`);
```

Comment:
This script should be a bash script, not a JS script.

Reply:
done.